library/Fission/CLI/IPFS/Connect.hs

```diff
@@ -7,7 +7,7 @@ module Fission.CLI.IPFS.Connect
 import           Fission.Prelude
 
 import qualified System.Console.ANSI as ANSI
-import           Data.List.NonEmpty as NonEmpty hiding ((<|))
+import           RIO.NonEmpty as NonEmpty hiding ((<|))
```

Comment:
We prefer RIO right?

Reply:
Yes 👍 We probably didn’t realize that it was in there at the time

A lot of this code is _pretty old_ 😅 It’s actually pretty great that it’s mostly just worked all this time (IPFS annoyances aside)